.travis.yml

```diff
@@ -7,16 +8,21 @@ matrix:
     env: CXX=g++-4.8
   - node_js: 8
     env: CXX=g++-4.8
+    #  - node_js: stable
+    #    env: CXX=g++-4.8
 
 script:
   - npm run lint
   - npm run test
-  - make test
```

Comment:
Why remove this @VictorBjelkholm ?

Reply:
Good catch @JonKrone ref: https://github.com/ipfs/js-ipfs/pull/1181#issuecomment-360285065

Reply:
@diasdavid this is coming from the file which is in ipfs/ci-sync, that was taken from js-ipfs-api as you mentioned that was the correct one we want to have in every repository.

Reply:
@VictorBjelkholm it was the base. In js-ipfs we also have sharness tests which get run through that makefile. We talked about extracting the sharness tests to a separate repo, are you onto that?